### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/accessibility#103]
Results can be recreated using HTML_Codesniffer \n

## Results for https://pages.18f.gov/accessibility/:
* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.3:1. Recommendation: change text colour to #0476d2.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > a
 * `<a class="skip-link visuallyhidden focusable" href="#main">Skip to Main Content</a>`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * #resources > a
 * `<a class="anchorjs-link " href="#resources" aria-label="Anchor link for: resources" data-anchorjs-icon="" style="opacity: 1; font-family: anchorjs-icons; font-style: normal; font-variant: normal; font-weight: normal; line-height: 1; padding-left: 0....`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * #sites-tutorials > a
 * `<a class="anchorjs-link " href="#sites-tutorials" aria-label="Anchor link for: sites tutorials" data-anchorjs-icon="" style="opacity: 1; font-family: anchorjs-icons; font-style: normal; font-variant: normal; font-weight: normal; line-height: 1; padd...`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * #tools > a
 * `<a class="anchorjs-link " href="#tools" aria-label="Anchor link for: tools" data-anchorjs-icon="" style="opacity: 1; font-family: anchorjs-icons; font-style: normal; font-variant: normal; font-weight: normal; line-height: 1; padding-left: 0.375em;">...`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(1) > a:nth-child(1)
 * `<a href="https://18f.gsa.gov">18F</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(1) > a:nth-child(2)
 * `<a href="https://github.com/18F/accessib...">https://github.com/18F/accessibility</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(2) > a:nth-child(1)
 * `<a href="https://github.com/18F/pages/">18F Pages</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(2) > a:nth-child(2)
 * `<a href="https://github.com/18F/guides-template/">18F Guides theme</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(2) > a:nth-child(3)
 * `<a href="https://github.com/cfpb/docter/">DOCter</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(2) > a:nth-child(4)
 * `<a href="http://cfpb.github.io/">CFPB</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(3) > a:nth-child(1)
 * `<a href="https://github.com/18F/accessibility/edit/18f-pages/index.md">Edit this page</a>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * html > body > div > footer > div > p:nth-child(3) > a:nth-child(2)
 * `<a href="https://github.com/18F/accessibility/issues">file an issue</a>`

## Summary:
* 12 Errors
